### DISCUSSION TITLE
建议PCL2在第一次启动时以当前系统显示语言作为启动器显示语言

### DISCUSSION THREAD

**Discussion by zzr-shift** [Meloong-Git/PCL#4599] [提问] [answered]
对他人更友好qwq

**Comment by SALTWOOD** [comment, marked as answer]
多语言没有正式加入，目前只有简体中文
多语言加入的时候肯定会做一些检测，或者是在第一次启动的时候要求选择的

**Comment by 3gf8jv4dv** [comment]
The `main` branch currently only supports Simplified Chinese.
In #4145, the system display language is now detected when the launcher is first opened.

`main` 分支目前只有简体中文。
在 #4145 中，目前已经会在首次打开启动器时检测系统显示语言。
